lib/class-wp-rest-blocks-renderer-controller.php

```diff
@@ -0,0 +1,144 @@
+				'name' => array(
+					'description' => __( 'Unique registered name for the block.', 'gutenberg' ),
+					'type'        => 'string',
+				),
+			),
+			array(
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'get_item_output' ),
+				'permission_callback' => array( $this, 'get_item_output_permissions_check' ),
+				'args'                => array(
+					'context' => $this->get_context_param( array( 'default' => 'view' ) ),
+				),
+			),
+			'schema' => array( $this, 'get_public_item_schema' ),
+		) );
+	}
+
+	/**
+	 * Checks if a given request has access to read blocks.
+	 *
+	 * @since ?
+	 * @access public
+	 *
+	 * @return true|WP_Error True if the request has read access, WP_Error object otherwise.
+	 */
+	public function get_item_output_permissions_check() {
+		if ( ! current_user_can( 'edit_posts' ) ) {
+			return new WP_Error( 'gutenberg_block_cannot_read', __( 'Sorry, you are not allowed to read Gutenberg blocks as this user.', 'gutenberg' ), array(
+				'status' => rest_authorization_required_code(),
+			) );
+		}
+
+		return true;
+	}
+
+	/**
+	 * Returns block output from block's registered render_callback.
+	 *
+	 * @since ?
+	 * @access public
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 */
+	public function get_item_output( $request ) {
+		if ( ! isset( $request['name'] ) ) {
+			return new WP_Error( 'rest_block_invalid_name', __( 'Invalid block name.', 'gutenberg' ), array( 'status' => 404 ) );
+		}
+
+		$registry = WP_Block_Type_Registry::get_instance();
+		$block    = $registry->get_registered( $request['name'] );
+
+		if ( ! $block || ! $block instanceof WP_Block_Type ) {
+			return new WP_Error( 'rest_block_invalid_name', __( 'Invalid block name.', 'gutenberg' ), array( 'status' => 404 ) );
+		}
+
+		$atts = $this->prepare_attributes( $request->get_params() );
+
+		$data = array(
+			'output' => $block->render( $atts ),
+		);
+		return rest_ensure_response( $data );
+	}
+
+	/**
+	 * Fix potential boolean value issues. The values come as strings and "false" and "true" might generate issues if left like this.
+	 *
+	 * @param array $attributes Attributes.
+	 * @return mixed Attributes.
+	 */
+	public function prepare_attributes( $attributes ) {
+		foreach ( $attributes as $key => $value ) {
+			if ( 'false' === $value ) {
+				$attributes[ $key ] = false;
+			} elseif ( 'true' === $value ) {
+				$attributes[ $key ] = true;
+			}
+		}
+
+		return $attributes;
+	}
+
+	/**
+	 * Retrieves block's output schema, conforming to JSON Schema.
+	 *
+	 * @since ?
+	 * @access public
+	 *
+	 * @return array Item schema data.
+	 */
+	public function get_item_schema() {
+		return array(
+			'$schema'    => 'http://json-schema.org/schema#',
+			'title'      => 'blocks-renderer',
+			'type'       => 'object',
+			'properties' => array(
+				'output' => array(
```

Comment:
I think the `rendered` name would be more in-line with the rest of the REST API.

Reply:
Done in c4abc69